### PR TITLE
Install in userguide

### PR DESCRIPTION
**Pull request by heckj**
restructuring some of the top level structure and cleanup of errors in documentation generation

### CODE REVIEW

**Comment by heckj** [comment]
@RackHD/corecommitters @RackHD/doccommitters please review

File: docs/rackhd/secure_erase.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Disk Secure Erase Workflow Support
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~!~~~~~~~
```
Comment (DavidjohnBlodgett) [inline docs/rackhd/secure_erase.rst]: @heckj was the "!" expected?

**Comment by benbp** [comment]
👍

**Comment by DavidjohnBlodgett** [comment]
👍 with a minor issue/question.

File: docs/rackhd/secure_erase.rst (hunk 1 above)
Comment (heckj) [inline docs/rackhd/secure_erase.rst]: Whoops!! Good catch, thank you!

**Pull request merged**